service/src/main/kotlin/fi/espoo/evaka/shared/async/AsyncJob.kt

```diff
@@ -79,6 +79,15 @@ sealed interface UrgentAsyncJob : AsyncJobPayload {
     ) : UrgentAsyncJob {
         override val user: AuthenticatedUser? = null
     }
+
+    data class MarkMessageAsSent(
```

Comment:
One field need to be added to the job payload. Is it better to add a new AsyncJob or add a nullable parameter to the existing one? 🤷